prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,29 @@
+"""
+CP1404 - GUI program for dynamic labels
+Jarrod Eaton
+12/11/2023
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+
+
+class DynamicLabelsApp(App):
+    """A Kivy application that dynamically creates a label for each name in a predefined list."""
+    def __init__(self, **kwargs):
+        """Initialise the app with a list of names."""
+        super().__init__(**kwargs)
+        self.names_list = ["Alice", "Bob", "Charlie", "Diana"]
+
+    def build(self):
+        """Build the app's GUI by loading the Kivy layout file and creating label widgets
+        for each name in the list."""
+        self.root = Builder.load_file('dynamic_labels.kv')
+        for name in self.names_list:
```

Comment:
Implementation here is fine, but would it be better to move the creating of the labels into it's own function?